rate-limiting.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Rate limiting
+description: "What is rate limiting and how to deal with it."
+icon: "gauge-max"
+---
+
+Whenever the octomind agent is discovering your page or the test runner is executing test cases it is sending requests requests to your page (or your app).
+
+To ensure a well defined behavior all requests are send through one fixed IP address: `35.192.162.70`
+
+## Agent discovery
+
+Agent discovery happens as if a human would use your web application, most of the time the agent interacts with the website slightly faster, than a human would
+and there will more than one discovery run in parallel.
+
+## Test runner
+
+When you do a "run all" or a scheduled run on a given test target all activated test cases are run at once and with up to 20 parallel browser sessions.
+This can lead to either an overload of your application or a rate limiter getting active, if you have one in place.
+
+## Login / Authentication
+
+If your application requires login the configured test user is used for the agent discovery as well as for all the test runs. This could also lead to 
+"hitting a limit of allowed logins", depending on your login system. If your are using a 3rd party login service, there are most likely security configurations
+in place than prevent users from logging to frequently.
+
+### OTP / 2FA
+
+While octomind ist supporting a login with a one time password (OTP), one of the most likely limits that a test run will hit, is how often the same OTP code 
+can be used for login.
+
+## Rate limiter
+
+If your application is protected against [DoS Attacks](https://en.wikipedia.org/wiki/Denial-of-service_attack) with a
+[rate limiter](https://en.wikipedia.org/wiki/Rate_limiting) it is very likely that octomind test runs and/or discoverys will fail.
+The easiest solutions is to configure and exception for our IP address: `35.192.162.70` (white listing).
```

Comment:
same comment about linking to the IP adress section here

Reply:
comes in the next PR :))))

Reply:
working on it